app/app.rb

```diff
@@ -47,13 +46,13 @@ def run_available_options(choice)
 
   def main_section
     'Please choose an option by entering a number:
-1 - List all books
-2 - List all people
-3 - Create a person
-4 - Create a book
-5 - Create a rental
-6 - List all rentals for a given person
-7 - Exit'
+  1 - List all books
+  2 - List all people
+  3 - Create a person
+  4 - Create a book
+  5 - Create a rental
+  6 - List all rentals for a given person
+  7 - Exit'
   end
```

Comment:
- [ ] Based on the requirement:  
`Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).`

- [ ] It would be better to move all the input questions/options back to the main.rb